tools/ratbagctl.test.in

```diff
@@ -462,23 +462,23 @@ class TestRatbagCtlLED(TestRatbagCtl):
     def test_led_n_get(self):
         self.setProfile(0)
         r = self.launch_good_test("led 0 get test_device")
-        self.assertEqual(r, "LED: 0 type: logo, depth: rgb, mode: off")
+        self.assertEqual(r, "LED: 0 type: logo, depth: rgb888, mode: off")
```

Comment:
Looking at this, I do not think we should change the output of ratbagctl here. RGB usually means 8bits, so we should be fine.

Having a test that uses rgb111 would be interesting too given that I don't think the current ratbagctl code will be working properly (I think that it will accept anything between  and ffffff, while only 1 bits values should be good).

This can be added in a later patch though